.gitmodules

```diff
@@ -84,7 +84,7 @@
 	url = git@github.com:Devsh-Graphics-Programming/Nabla-Continous-Integration-Python-Framework.git
 [submodule "3rdparty/boost/superproject"]
 	path = 3rdparty/boost/superproject
-	url = ../boost.git
+	url = git@github.com:Devsh-Graphics-Programming/boost.git
```

Comment:
as @AnastaZIuk will tell you, this needs to stay as it was, you need to for the submodules yourself as well.

Its important the submodules are relative.